src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -259,15 +259,15 @@ def get_keyring_with_caps(self, entity: AuthEntity, caps: List[str]) -> str:
                 'entity': entity,
                 'caps': caps,
             })
-            if ret:
+            if err:
```

Comment:
@adk3798: could you please take a look on this fixup?

Reply:
I think what you changed looks correct, checking err has been our standard way of checking these commands' success so far.